pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -396,7 +393,7 @@ func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
 		obj.HairpinMode = PromiscuousBridge
 	}
 	if obj.EvictionHard == nil {
-		temp := "memory.available<100Mi"
+		temp := "memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%"
```

Comment:
For storage I think performance is relative to available percentage of capacity and not an absolute value. so a percentage here makes sense